Gordon360/Services/ProfileService.cs

```diff
@@ -63,7 +65,56 @@ public AlumniProfileViewModel GetAlumniProfileByUsername(string username)
                 result = alumni;
             return result;
         }
+        /// <summary>
+        /// get advisors for particular student
+        /// </summary>
+        /// <param name="id">student id</param>
+        /// <returns></returns>
+        public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
+        {
+            //var query = RawSqlQuery<ADVISOR_SEPARATE_Result>.query("ADVISOR_SEPARATE @ID", new SqlParameter("ID", SqlDbType.VarChar) { Value = id });
+            var query = _unitOfWork.AccountRepository.FirstOrDefault(x => x.gordon_id == id);
+            var idParam = new SqlParameter("@ID", id);
+            // Stored procedure returns row containing advisor1 ID, advisor2 ID, advisor3 ID 
+            var idResult = RawSqlQuery<ADVISOR_SEPARATE_Result>.query("ADVISOR_SEPARATE @ID", idParam).FirstOrDefault();
+
+            //Object Array to store advisors information
+            AdvisorViewModel[] advisorsID = new AdvisorViewModel[3];
```

Comment:
The rest of this code appears to be more complicated than necessary, so I'm suggesting another way to write it.  You may need to fix up the indentation.